frontend/packages/ceph-storage-plugin/integration-tests-cypress/support/index.ts

```diff
@@ -92,7 +95,14 @@ Cypress.Commands.add('install', (mode: 'Internal' | 'Attached' = 'Internal', enc
             expect(item.hasOwnProperty('cluster.ocs.openshift.io/openshift-storage')).toBeTruthy(),
           );
       });
-
+      // check if taint is applied
+      cy.log('verifying taint is applied to worker nodes');
+      cy.exec('oc get nodes -l "node-role.kubernetes.io/worker" -o json').then((res) => {
+        const { items } = JSON.parse(res.stdout);
+        items.forEach((item) =>
+          expect(item.spec.taints[0].key).toEqual('node.ocs.openshift.io/storage'),
+        );
```

Comment:
```suggestion
        items.forEach((item) =>
          expect(item.spec.taints.some(taint => Object.key(taint) === 'node.ocs.openshift.io/storage' && Object.value(taint) === true)).toBeTruthy()
        );
```
We can't assume that the first taint would indeed be the OCS taint. The node could be tainted with other taints too.